docs/tutorials/enterprise_beginner/2_prepare_the_landing_page.md

```diff
@@ -42,7 +42,7 @@ Preparing a new layout requires three things:
 
 #### Create entry in configuration
 
-First create a new file for layout configuration, `app/config/layouts.yml`:
+First create a new file for layout configuration, `app/config/layouts.yaml`:
```

Comment:
File path is changed right now but it looks like tutorial app has to be migrated to Symfony 4 as well. There should be no `app/config/layouts.yaml` file, it most likely has to be moved to `config/packages/ezplatform_page_builder.yaml`

Reply:
All tutorials will be verified and modified after beta, but extentions have been changed in this PR for simplicity